docs-website/package.json

```diff
@@ -14,9 +14,9 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.3.0",
-    "@docusaurus/preset-classic": "2.3.0",
-    "@docusaurus/theme-mermaid": "^2.3.0",
+    "@docusaurus/core": "^2.3.1",
```

Comment:
we can bump deps, but renovate will pin them again so can we drop the `^`?

Reply:
Yep. Didn't notice it added them back. Fixed.